### PR TITLE
Updated GasFeeController to latest package version

### PR DESCRIPTION
**Pull request by shanejonas**
## **Description**
This bumps `GasFeeController` to the latest package version

### CODE REVIEW

**Comment by shanejonas** [comment]
@metamaskbot update-policies

**Comment by shanejonas** [comment]
@metamaskbot update-policies

**Comment by adonesky1** [comment]
Looks like this bump requires that `@metamask/network-controller` be bumped to v17.2.0 per the changelog: 
https://github.com/MetaMask/core/blob/main/packages/gas-fee-controller/CHANGELOG.md

**Comment by shanejonas** [comment]
@metamaskbot update-policies

**Review by adonesky1 — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -276,7 +276,7 @@
     "@metamask/message-manager": "^7.3.0",
     "@metamask/metamask-eth-abis": "^3.0.0",
     "@metamask/name-controller": "^4.2.0",
-    "@metamask/network-controller": "^17.1.0",
+    "@metamask/network-controller": "17.2.0",
```
Comment (adonesky1) [inline package.json:279]:
```suggestion
    "@metamask/network-controller": "^17.2.0",
```

**Review by adonesky1 — APPROVED** [review]

**Review by legobeat — APPROVED** [review]

**Pull request merged**